flake.nix

```diff
@@ -0,0 +1,47 @@
+{
+  description = "Ethereum network crawler, API, and frontend";
```

Comment:
This is part of a build system that I use: https://github.com/NixOS/nixpkgs. I didn't realize I committed the files. It makes it nicer for people who want to install and run the project, and for those who want to develop it because there is a shared development environment. It's up to you if you want to keep it. I will make it for my own use, and keep it uncommitted if it's not something you would like to have in the project.

I see there's also a project to run Ethereum projects with nix: https://github.com/nix-community/ethereum.nix
There's pretty cool. I should give it a shot. I never thought to look for it.

Reply:
+1 flake for nix deployment is very much appreciated!